packages/delegation-revoke-archiver/src/delegationRevokeArchiverConsumerServiceV2.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  CorrelationId,
+  DelegationEventEnvelopeV2,
+  generateId,
+  missingKafkaMessageDataError,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import {
+  // getInteropHeaders,
+  logger,
+  RefreshableInteropToken,
+} from "pagopa-interop-commons";
+// import { getInteropBeClients } from "./clients/clientsProvider.js";
+
+// const { agreementProcessClient, purposeProcessClient } = getInteropBeClients();
```

Comment:
Same for the clients, let's set them up in the index and pass them as params to handleMessage